source/edge-server/permissions.txt

```diff
@@ -0,0 +1,266 @@
+data in Atlas and the data that a client can read and write.
+
+In a healthcare setting similar to the Edge server instance example above, the
+Edge Server instance may only read and write data that is relevant to its own 
+facility. A client may then further restrict the permissions. For example, a 
+patient in the facility may only view their own data.
+
+You represent these as independent roles within the ``roles`` array.
+For example:
+
+.. code-block:: json
+
+   {
+     "collection": "Visits",
+     "database": "PatientRecords",
+     "roles": [
+       {
+         "name": "facilityItemsOnly",
+         "apply_when": { 
+           "%%user.type": {
+             "$eq": "edge"
+           }
+         },
+         "document_filters": {
+           "write": {
+             "facility_id": "%%user.id"
+           },
+           "read": {
+             "facility_id": "%%user.id"
+           },
+         },
+         "insert": true,
+         "delete": true,
+         "search": true,
+         "read": true,
+         "write": true
+       },
+       {
+         "name": "patientOwnRecordsOnly",
+         "apply_when": {},
+         "document_filters": {
+           "write": {
+             "patient_id": "%%user.id"
+           },
+           "read": {
+             "patient_id": "%%user.id"
+           }
+         },
+         "read": true,
+         "write": true,
+         "insert": true,
+         "delete": true,
+         "search": true
+       }
+     ]
+   }
+
+In this example, a ``PatientRecords`` database has a ``Visits`` collection.
+The two roles determine which documents the Edge Server instance can sync,
+and which documents an individual connected client can sync:
+
+- The ``facilityItemsOnly`` role filters items that sync to the Edge Server
+  instance. The only items that sync to the facility are records where the
+  ``facility_id`` matches the Edge Server instance ``user_id``.
+- The ``patientOwnRecordsOnly`` role filters items that can sync to a connected
+  client device. The only items that a connected client can see are visits 
+  where the ``patient_id`` is the client's ``user_id``.
+
+In a setting where you have many different types of clients that should each
+have access to a subset of the data, you may define many different client 
+roles. For example, in a healthcare setting, you may use different roles for:
+
+- Patients: provide access to their own medical records *and* address billing 
+  information.
+- Billing specialists: provide access to patient address and billing 
+  information, but not medical records
+- Doctors: provide access to patient medical records, but not address and
+  billing information.
+
+.. figure:: /images/field-level-permissions.png
+   :alt: Documents showing different data available to three different roles - a patient, a billing specialist, and a doctor.
+   :lightbox:
+
+Edge Server Supports Field-Level Permissions
```

Comment:
```suggestion
Field-Level Permissions
```
since we've only been discussing what Edge Server supports, this seems unnecessary